conf/shh.config

```diff
@@ -29,6 +29,10 @@ process {
     }
 }
 
+conda {
```

Comment:
Hm, no. We need to have different profiles then for Jena if you guys want to have both .... or you need to specify this in a different profile - I personally would love to keep using Singularity as this is more stable.